enhancements/ingress/transition-ingress-from-beta-to-stable.md

```diff
@@ -0,0 +1,302 @@
+---
+title: transition-ingress-from-beta-to-stable
+authors:
+  - "@Miciah"
+reviewers:
+  - "@candita"
+  - "@danehans"
+  - "@frobware"
+  - "@knobunc"
+  - "@miheer"
+  - "@rfredette"
+  - "@sgreene570"
+approvers:
+  - "@danehans"
+  - "@frobware"
+  - "@knobunc"
+creation-date: 2021-03-09
+last-updated: 2021-03-09
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Transitioning Ingress API from Beta to Stable
+
+This enhancement transitions OpenShift fully onto the stable
+`networking.k8s.io/v1` API version of the Ingress API.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement updates the
+[ingress-to-route](https://github.com/openshift/openshift-controller-manager/blob/master/pkg/route/ingress/ingress.go)
+controller to use the stable `networking.k8s.io/v1` version of the Ingress API.
+Although the API server already supports the v1 API version without this
+enhancement, the ingress-to-route controller requires changes to transition from
+using the v1beta1 client to using the v1 client and to support new v1 features.
+These new features include the `spec.pathType` and `spec.ingressClassType`
+fields as well as the associated IngressClass API.  This enhancement does *not*
+extend the Route API to accommodate new features in the Ingress API; these are
+only supported to the extent that they are compatible with the Route API.
```

Comment:
Does "these" in L48 refer to the "new features in the Ingress API" ?  If so, it would be useful to have a table of those that are compatible with the existing Route API and those that are not.  This would help describe goal #1 below better.

Reply:
It does.  I'll add a table.  Thanks for the suggestion!